src/environments/environment.production.ts

```diff
@@ -14,4 +14,7 @@ export const environment: Partial<BuildConfig> = {
     enableSearchComponent: false,
     enableBrowseComponent: false,
   },
+  info: {
+    enableCookieConsentPopup: true,
+  },
```

Comment:
@AndreaBarbasso : I see the e2e tests are still failing from the Orejime popup getting in the way.  I think it might be that these `environment.*.ts` settings should be **removed**.  

I think the *default* value of `enableCookieConsentPopup` should be added to the `default-app-config.ts` alongside all the other default values.  Otherwise, I *think* the `environment.*.ts` settings might *override* the settings elsewhere based on how our configs are all merged together here: https://github.com/DSpace/dspace-angular/blob/main/src/config/config.util.ts#L18-L21